src/main/java/pl/put/poznan/transformer/rest/TextTransformerController.java

```diff
@@ -35,22 +35,25 @@ public Response get(@PathVariable String text,
 
         // do the transformation, you should run your logic here, below just a silly example
         TextTransformerInterface transformer = null;
-        for(String transform: transforms) {
-            if(transform.equals("upper"))
+        for (String transform : transforms) {
+            if (transform.equals("upper"))
```

Comment:
Zmień to proszę na else if albo switch(case). Bez sensu sprawdzać za każdym razem + będzie czytelniej

Reply:
Zajmę się tym albo zaraz albo jutro :) i dodam również rzeczy odpowiedzialne za generowanie dokumentacji w tych klasach